src/js/filter.js

```diff
@@ -0,0 +1,39 @@
+/* global document, window */
+
+const setUpFilter = (markerGroup, citiesToMarkers, data) => {
+  const filter = document.querySelector(".filter--scope");
+  let priorSelection = [
+    "Regional",
+    "Citywide",
+    "City Center",
+    "Transit Oriented",
+    "Main Street",
+  ];
+
+  const onScopeFilterChange = (markerGroup, citiesToMarkers, data) => () => {
+    const checked = document.querySelectorAll(".filter--scope :checked");
+    const selected = [...checked].map((option) => option.value);
```

Comment:
Does this work? I personally prefer this type of "inlining" because all the values are highly related to each other. No need for two distinct variables: checked and selected are synonyms!

You'll need to autoformat this code and update references to use checked rather than selected.

```suggestion
    const checked = document.querySelectorAll(".filter--scope :checked").map((option) => option.value);
```